meshroom/pipelines/hdrFusion.mg

```diff
@@ -0,0 +1,99 @@
+{
+    "header": {
+        "nodesVersions": {
+            "Publish": "1.3",
+            "LdrToHdrMerge": "4.1",
+            "CameraInit": "9.0",
+            "LdrToHdrSampling": "4.0",
+            "PanoramaPrepareImages": "1.1",
+            "LdrToHdrCalibration": "3.0"
+        },
+        "releaseVersion": "2023.2.0-develop",
+        "fileVersion": "1.1",
+        "template": true
+    },
+    "graph": {
+        "LdrToHdrMerge_1": {
+            "nodeType": "LdrToHdrMerge",
+            "position": [
+                800,
+                0
+            ],
+            "inputs": {
+                "input": "{LdrToHdrCalibration_1.input}",
+                "response": "{LdrToHdrCalibration_1.response}",
+                "userNbBrackets": "{LdrToHdrCalibration_1.userNbBrackets}",
+                "byPass": "{LdrToHdrCalibration_1.byPass}",
+                "channelQuantizationPower": "{LdrToHdrCalibration_1.channelQuantizationPower}",
+                "workingColorSpace": "{LdrToHdrCalibration_1.workingColorSpace}"
+            }
+        },
+        "LdrToHdrCalibration_1": {
+            "nodeType": "LdrToHdrCalibration",
+            "position": [
+                600,
+                0
+            ],
+            "inputs": {
+                "input": "{LdrToHdrSampling_1.input}",
+                "samples": "{LdrToHdrSampling_1.output}",
+                "userNbBrackets": "{LdrToHdrSampling_1.userNbBrackets}",
+                "byPass": "{LdrToHdrSampling_1.byPass}",
+                "calibrationMethod": "{LdrToHdrSampling_1.calibrationMethod}",
+                "channelQuantizationPower": "{LdrToHdrSampling_1.channelQuantizationPower}",
+                "workingColorSpace": "{LdrToHdrSampling_1.workingColorSpace}"
+            }
+        },
+        "LdrToHdrSampling_1": {
+            "nodeType": "LdrToHdrSampling",
+            "position": [
+                400,
+                0
+            ],
+            "inputs": {
+                "input": "{PanoramaPrepareImages_1.output}"
+            }
+        },
+        "CameraInit_1": {
+            "nodeType": "CameraInit",
+            "position": [
+                0,
+                0
+            ],
+            "inputs": {
+                "allowedCameraModels": [
+                    "pinhole",
+                    "radial1",
+                    "radial3",
+                    "brown",
+                    "fisheye1",
+                    "3deanamorphic4",
+                    "3deradial4",
+                    "3declassicld"
+                ]
+            }
+        },
+        "PanoramaPrepareImages_1": {
```

Comment:
No need for panoramaPrepareImages in this case

Reply:
Done.